docs/source/tutorial/tutorial_template.md

```diff
@@ -49,7 +49,7 @@ pytest-mock>=1.7.1, <2.0
 pytest~=7.2
 
 # Kedro dependencies and datasets to work with different data formats (including CSV, Excel, and Parquet)
-kedro~=0.18.10
+kedro~=0.18.11
```

Comment:
Nit: I'm not familiar with the release process, but I think it's a bit weird/bad practice to bump the lower bound each time if we're not introducing something that wouldn't otherwise work? Otherwise, perhaps we should just hard-pin, if the goal is to make sure user end up with exactly the new version.

Reply:
(Probably not a priority to discuss now. :))

Reply:
Indeed this is not necessary, equally we didn't bump `kedro-datasets`~=1.1 to ~=1.4